src/gui/activitywidget.cpp

```diff
@@ -556,7 +556,7 @@ void ActivitySettings::slotShowIssueItemCount(int cnt)
 {
     QString cntText = tr("Not Synced");
     if( cnt ) {
-        cntText = tr("Not Synced - %n file(s)", "", cnt);
+        cntText += QLatin1String(" (") + QString::number(cnt) + QLatin1String(")");
```

Comment:
is there not lanquages in which the  parentheses should be put before?

perhaps it's better to keep it in the tr()